tests/foreman/api/test_provisioning.py

```diff
@@ -22,12 +22,13 @@
 from wait_for import wait_for
 
 
+@pytest.mark.parametrize("provisioning_host", ['bios', 'uefi'], indirect=True)
 @pytest.mark.on_premises_provisioning
 @pytest.mark.rhel_ver_match('[^6]')
```

Comment:
Non-blocking, should we go for more `rhel` versions to be tested for pxe.

Reply:
This already covers rhel7, rhel8, rhel9